pkg/storaged/block/format-dialog.jsx

```diff
@@ -450,6 +458,10 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
                     partition_type = "21686148-6449-6e6f-744e-656564454649";
                 }
 
+                if (type == "swap") {
+                    partition_type = block_ptable.Type == "dos" ? "0x82" : "0657fd6d-a4ab-43c4-84e5-0933c84b4f4f";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19843-20240116-134034-00b55033-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)